tethysapp/app_store/git_install_handlers.py

```diff
@@ -47,7 +47,7 @@ def run_pending_installs():
     time.sleep(10)
     logger.info("Checking for Pending Installs")
 
-    app_workspace = app.get_app_workspace()
+    app_workspace = get_app_workspace(app)
```

Comment:
FYI. We have added the `app.get_app_workspace()` method back to the app class. Just make sure you are on the latest version of Tethys 4.